### PR TITLE
added: quit all and saveAll then quit commands, respectively:qa and waq

### PR DESCRIPTION
**Pull request by caiocutrim**
### I've added quitAll (:qa) and saveAllAndQuit(:waq) commands that I miss in project :+1:

### CODE REVIEW

File: lib/ex.coffee [hunk 1]
```diff
@@ -74,8 +74,13 @@ class Ex
   quit: ->
     atom.workspace.getActivePane().destroyActiveItem()
 
+  quitAll: ->
```
Comment (jazzpi) [inline lib/ex.coffee]: This isn't actually a vim command - there is `quitall` though.

File: lib/ex.coffee [hunk 2]
```diff
@@ -177,10 +182,15 @@ class Ex
     @write(args...).then => @quit()
 
   xit: (args...) => @wq(args...)
-
-  wa: ->
+  saveAll: ->
```
Comment (jazzpi) [inline lib/ex.coffee]: This also isn't a vim command, I think the only thing close is `wall`.

File: lib/ex.coffee [hunk 3]
```diff
@@ -199,9 +209,9 @@ class Ex
   substitute: (range, args) ->
     args = args.trimLeft()
     delim = args[0]
-    if /[a-z1-9\\"|]/i.test(delim)
+    if /[a-z]/i.test(delim)
```
Comment (jazzpi) [inline lib/ex.coffee]: Why did you revert this?

File: lib/ex.coffee (hunk 1 above)
Comment (caiocutrim) [inline lib/ex.coffee]: yes, it is
http://vimdoc.sourceforge.net/htmldoc/editing.html

File: lib/ex.coffee (hunk 3 above)
Comment (caiocutrim) [inline lib/ex.coffee]: I did'nt make this XD, trust me, I just create the :waq command as you can see here
http://vimdoc.sourceforge.net/htmldoc/editing.html

File: lib/ex.coffee (hunk 1 above)
Comment (jazzpi) [inline lib/ex.coffee]: Yes, `quitall` exists, but `quitAll` doesn't.

File: lib/ex.coffee (hunk 3 above)
Comment (jazzpi) [inline lib/ex.coffee]: Huh, looks like you worked off of code that didn't include #102. Maybe try pulling the repository again?

File: lib/ex.coffee (hunk 1 above)
Comment (caiocutrim) [inline lib/ex.coffee]: **quitAll** is function that I did reference on :qa shutcut

File: lib/ex.coffee [hunk 4]
```diff
@@ -74,8 +74,13 @@ class Ex
   quit: ->
     atom.workspace.getActivePane().destroyActiveItem()
 
+  quitAll: ->
+    atom.workspace.getPanes()[0].destroy()
+
   q: => @quit()
 
+  qa: => @quitAll()
+
```
Comment (caiocutrim) [inline lib/ex.coffee]: Here :)

File: lib/ex.coffee (hunk 1 above)
Comment (caiocutrim) [inline lib/ex.coffee]: so wy when I type :quitall the editor show a message that this command does not exit?

File: lib/ex.coffee (hunk 3 above)
Comment (caiocutrim) [inline lib/ex.coffee]: okay dude :)

**Comment by jazzpi** [comment]
Closing this because #120 is merged. Thanks again :smile:

**Pull request closed without merging**